### DISCUSSION TITLE
Authorization to use images from Digital in my book

### DISCUSSION THREAD

**Discussion by RicardoKers** [hneemann/Digital#1168] [General]
Hello, I'm Ricardo Kerschbaumer, professor of Digital Electronics in Brazil and I'm writing a book on the subject. For this book I need to draw several digital circuits and I would like to use Digital software to do these drawings. So, I'm writing here to ask if Mr Helmut Neemann authorizes me to publish the images generated in his software. If allowed, I would also like to publish images of the Digital interface, to show the readers of the book how this simulator works.
Thank you for your attention.

**Comment by hneemann** [comment]
@RicardoKers 
Feel free to use images or screenshots created with the simulator. 
The best quality is provided by the SVG export in combination with Inkscape and LaTeX.

**Reply by RicardoKers** [reply]
Thank you very much, it will be a great help.
